packages/core/test/preview.spec.ts

```diff
@@ -51,6 +82,7 @@ const expectedFunctionCallOne: FunctionCallActionInput = {
       myFunctionVar: 'myFunctionValue',
     },
     address: '0x' + '11'.repeat(20),
+    value: parseUnits('0.1', 'ether').toString(),
```

Comment:
It's obviously also possible to transfer value during a normal function call, and this tests that is properly handled in the preview.